extensions/wikia/FacebookClient/FacebookClient.class.php

```diff
@@ -369,47 +369,6 @@ public function logout() {
 	}
 
 	/**
-	 * Check if we should redirect back to the specified page by comparing it to this black list
```

Comment:
This code was duplicate functionality.  Consolidated into the UserLoginHelper.class.php method, `getRedirectUrl()`